src/components/meeting.jsx

```diff
@@ -85,12 +100,21 @@ export default class Meeting extends Component {
 							<div className="list-group-item">
 								<h5>{meeting.location}</h5>
 								<p className="my-0 mt-1">{meeting.formatted_address}</p>
-								<p className="my-0 mt-1">Other meetings at this address:</p>
+							</div>
+							<div className="list-group-item">
+								<h5>Other Meetings Here:</h5>
```

Comment:
All English language strings belong in Settings so they can be translated. I'm thinking of also making this "All Meetings" because it's looks a little weird to exclude just the one especially when there are a lot of meetings.